asciidoc/volume3/tf3-ch-8.3.2-biceps-content.adoc

```diff
@@ -146,6 +146,55 @@ In addition to specific device specializations (see <<vol3_clause_device_special
+
+NOTE: This implies that a MANUFACTURER that needs to use private codes has to keep a private coding system for these codes. Following the concept of private codes that is defined in <<ref_ieee_11073_10101_2020>>, a MANUFACTURER introduces and maintains a set of codes that adhere to the principles of the <<ref_ieee_11073_10101_2020>> nomenclature standards.
+
+There are different ways to define a globally unique identifier for the vendor-specific private coding system (e. g. URL, OID, etc.). In the context of medical device interoperability, the recommendation is to use  an "object identifier" (OID) for private codes as detailed in <<ref_ieee_11073_10101_2020>> and other purposes (e. g. private SDC extensions):
+
+* Compared with other identifiers (e.g. URLs) OIDs are very compact, which helps to keep the size of data messages low
+
+* It is free of charge to register a globally unique enterprise OID with https://www.iana.org/assignments/enterprise-numbers/assignment/apply/[IANA].
+
+An enterprise OID assigned by IANA always starts with the prefix *iso.org.dod.internet.private.enterprise (1.3.6.1.4.1)*. The actual enterprise node number, which follows the prefix, is globally unique to the medical device vendor (e. g. "_1.3.6.1.4.1.12345_" for the *Medical Device XYZ Ltd.* company).
+
+Once the enterprise OID is registered, it is up to the medical device vendor to extend the OID by adding child nodes. The child nodes don't have a globally unique meaning, but the enterprise OID plus the child nodes are required to be globally unique. It is recommended to define a child node schema, and assign unique numbers to the concepts on each node level.
+
+.OID Vendor-specific Child Node Schema
+====
+ * 1.3.6.1.4.1.12345.*<Namespace>*: this could be, for example, the business group id within the enterprise ("Patient Monitoring" = 1, "Ventilation" = 2, etc.).
+ * 1.3.6.1.4.1.12345.*<Namespace>*.*<Type>*: this could be, for example, the id of the type defined within the business group of the enterprise ("Private Code System" = 1, "SDC Extensions" = 2, etc.)
+* 1.3.6.1.4.1.12345.*<Namespace>*.*<Type>*.*<Version>*: it is highly recommended to define the version number as the last child node in the OID. Different versions indicate a major (maybe breaking) change to the *Type* defined by the parent node.
+====
+
+In the example above, the OID "_1.3.6.1.4.1.12345.2.1.1_" would represent a *private coding system* identifier in version *1* issued by the the *ventilation* business group of the *Medical Device XYZ Ltd.* company.
+
+.R0702
+[sdpi_requirement#r0702,sdpi_req_level=shall]
+****
+The companies claiming conformity with the IHE SDPi profile shall disclose their OIDs used for private codes as detailed in <<ref_ieee_11073_10101_2020>>, for private SDC extensions, and so on.
```

Comment:
```suggestion
The <<term_manufacturer>> shall disclose the OIDs that it uses throughout SDC communication.
```

- I think "claiming conformity with the IHE SDPi profile" is redundant as the manufacturer that conforms to SDPi is the one that has to fulfill this requirement anyway.
- " used for private codes as detailed in <<ref_ieee_11073_10101_2020>>. for private SDC extensions, and so on." this is not precise enough from my perspective. I would move the "private codes" and "SDC extensions" to the requirement's notes section. (e.g. "This includes private codes as detailed in <<ref_ieee_11073_10101_2020>>, private SDC extensions and private PKP OIDs)."
- I think we should also introduce the term OID.